torat_server/client_ishell.go

```diff
@@ -50,6 +49,12 @@ func (client activeClient) shellClient() {
 			Help:      "print the content of a file: usage cat <file>",
 			Completer: fileCompleter,
 		},
+		{
+			Name:      "shred",
+			Func:      client.Shred,
+			Help:      "remove a remote file by shreding it: usage shred <file>",
```

Comment:
remove traces of a file by overwriting it with random data and zeros and then deleting it.

Reply:
okay